_includes/scenarios/integrate/access-the-connector.md

```diff
@@ -1,16 +1,61 @@
 The primary integration capability of the Connector is the REST API. In order to use it, you should have received an API-Key for the respective Connector. An API-Key so far has all authorizations for accessing the API.
 
-## Interactive Documentation
+# API platform
 
-You can find the REST API documentation hosted on your Connector on the following HTTP endpoints. Swagger and Rapidoc are interactive websites hosted on the Connector with which you can try out the various APIs interactively.
+to use the api platform hosted on the connector you need to make the following config changes:
+
+1. the http server must be enabled in the configuration of the connector.
```

Comment:
Gehört das nicht in die operate Session a la "Setting up the Connector for development purposes"? Und hier ein Verweis auf dieses, dass man einen Development Connector benutzen sollte?